### PR TITLE
libxl: install and run xen-init-dom0 at boot time

### PR DESCRIPTION
**Pull request by jean-edouard**
OXT-916

Signed-off-by: Jed <lejosnej@ainfosec.com>

### CODE REVIEW

**Comment by crogers1** [comment]
+1.  What's libnl for?

**Comment by jean-edouard** [comment]
Dunno but OE says we use it, and complains if we don't have it in DEPENDS :)

**Comment by crogers1** [comment]
Ah, I see.  Also, we might need selinux rules for the init script and xen-init-dom0 binary itself.

**Comment by richpersaud** [comment]
libnl dependency may be related to Remus and COLO 

https://wiki.xen.org/wiki/COLO_-_Coarse_Grain_Lock_Stepping

https://wiki.xenproject.org/wiki/Xen_Project_Release_Features

Would be nice if we could disable those features in LibXL and eliminate the libnl dependency, but maybe that requires Kconfig and Xen 4.7.

**Comment by rossphilipson** [comment]
Build here (testing now):
http://openxt.ainfosec.com/builds/custom/master/584

File: recipes-openxt/xenclient/xenclient-dom0-tweaks/xenstore-init [hunk 1]
```diff
@@ -2,4 +2,3 @@
 
 # add xenstore entries here
 
-xenstore-write domid 0
```
Comment (rossphilipson) [inline recipes-openxt/xenclient/xenclient-dom0-tweaks/xenstore-init]: Why is this removed? Does xen-init-dom0 do that now?

Comment (jean-edouard) [inline recipes-openxt/xenclient/xenclient-dom0-tweaks/xenstore-init]: Yes
